support-frontend/assets/pages/digital-subscription-checkout/digitalSubscriptionCheckoutActions.js

```diff
@@ -45,6 +47,8 @@ const formActionCreators = {
   setBillingPeriod: (billingPeriod: DigitalBillingPeriod): Action => ({ type: 'SET_BILLING_PERIOD', billingPeriod }),
   setPaymentMethod: (paymentMethod: PaymentMethod) => (dispatch: Dispatch<Action>, getState: () => State) => {
     const state = getState();
+    storage.setSession('selectedPaymentMethod', paymentMethod);
```

Comment:
why do we have to put in storage? does all that get transferred into GA when the purchase happens?

Reply:
I've noticed that we send the payment method as a custom dimension on any events we send and that it comes from the session. We may actually not need to as it is a session level dimension in GA but contributions set it and it doesn't hurt so I think it's safer to.

Reply:
I thought that redux was the best place for state, is this needed in storage because it needs persist over page loads?